src/controllers/european.controller.ts

```diff
@@ -206,7 +206,7 @@ export class EuropeanController extends SessionController<EuropeBlueLinkyConfig>
           },
           body: {
             pushRegId: credentials.gcm.token,
-            pushType: 'GCM',
+            pushType: 'APNS',
```

Comment:
Not sure how this would work for non-ios devices 😬

Reply:
The old way gives an error, running bluelinky on a non-Android device :)